docs/zh_CN/tutorial/tutorial.md

```diff
@@ -0,0 +1,3 @@
+# NanoMQ使用场景教程
+
+本节为常用场景提供了教程。
```

Comment:
Is there anything more?

Reply:
Not now. But we can add tutorial here, if there is more to come.

Reply:
Good